.travis.yml

```diff
@@ -10,5 +10,14 @@ install:
   - npm install
 notifications:
   email: false
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - "export DISPLAY=:99.0"
+  - "wget http://selenium.googlecode.com/files/selenium-server-standalone-2.31.0.jar"
```

Comment:
yeah - it didn't work :( just want to get a running build and then ill refactor it to use the npm module.